fix(command): Fix and enhance /aqqbot send subcommand

Tsingloong611 opened the pull request:
This commit comprehensively fixes and enhances the `/aqqbot send` subcommand to make it fully functional and user-friendly.

- Fix: The command now correctly handles multi-word messages by properly parsing and joining arguments, resolving the critical bug where messages with spaces would fail.

- Feat: Added robust error handling. It now validates the group ID to prevent `NumberFormatException`

Tsingloong611 committed:
fix(command): Fix and enhance /aqqbot send subcommand

This commit comprehensively fixes and enhances the `/aqqbot send` subcommand to make it fully functional and user-friendly.

- Fix: The command now correctly handles multi-word messages by properly parsing and joining arguments, resolving the critical bug where messages with spaces would fail.

- Feat: Added robust error handling. It now validates the group ID to prevent `NumberFormatException`

- Feat: Implemented full tab completion support to guide users with prompts for "<群号>" and "<消息内容>", improving overall usability.

alazeprt commented on common/src/main/kotlin/top/alazeprt/aqqbot/command/AParentCommand.kt:
```diff
@@ -28,9 +29,10 @@ class AParentCommand(val plugin: AQQBot) : ACommand {
+            1 -> listOf("whitelist", "status", "help", "reload", "send")
             2 -> return when (args[0]) {
                 "whitelist" -> listOf("bind", "unbind", "reset", "query", "info")
+                "send" -> listOf("<groupId>")
```
I think we don't need this

Tsingloong611 replied:
removed

alazeprt commented on common/src/main/kotlin/top/alazeprt/aqqbot/command/AParentCommand.kt:
```diff
@@ -41,6 +43,7 @@ class AParentCommand(val plugin: AQQBot) : ACommand {
                     "reset" -> listOf("qq", "player")
                     else -> emptyList()
                 }
+                "send" -> listOf("<message>")
```
I think we don't need this

Tsingloong611 replied:
removed

alazeprt commented on common/src/main/kotlin/top/alazeprt/aqqbot/command/sub/SubSend.kt:
```diff
@@ -22,8 +22,13 @@ class SubSend(val plugin: AQQBot): ACommand {
+        val groupId = try {
+            args[1].toLong()
+        } catch (e: NumberFormatException) {
+            sender.sendMessage(Component.text("无效的群号: ${args[1]}", NamedTextColor.RED))
```
Move it into messages configuration

Tsingloong611 replied:
done

alazeprt commented:
If you want to implement full tab completion, please you implement all sub commands tab completion (ex. /aqqbot whitelist add)

alazeprt commented:
> If you want to implement full tab completion, please you implement all sub commands tab completion (ex. /aqqbot whitelist add)

Besides, you can add it to messages configuration (optional)

Tsingloong611 committed:
fix(command): Fix and enhance /aqqbot send subcommand

 - remove unnecessary tab completion.
 - move messages to the configuration file as requested.

alazeprt committed:
refactor: format SubSend.kt

alazeprt committed:
Update messages.yml
